galaxy/worker/importers/base.py

```diff
@@ -65,17 +66,26 @@ def make_content(self):
             raise exc.TaskError('Invalid name, only aplhanumeric characters, '
                                 '"-" and "_" symbols are allowed.')
 
-        obj, is_created = models.Content.objects.get_or_create(
-            namespace=ns,
-            repository=repo,
-            content_type=models.ContentType.get(self.data.content_type),
-            original_name=original_name,
-            defaults={
-                'name': self.translate_content_name(name),
-                'is_valid': False,
-            }
-        )
-
+        try:
+            # Check for an existing Content object matching name
+            is_created = False
+            obj = models.Content.objects.get(namespace=ns,
```

Comment:
Please try to fit your code to 80 characters line width.

Reply:
OK.